pkgs/os-specific/linux/kernel-headers/default.nix

```diff
@@ -90,12 +92,24 @@ let
         # `$(..)` expanded by make alone
         "HOSTCC:=$(CC_FOR_BUILD)"
         "HOSTCXX:=$(CXX_FOR_BUILD)"
+      ]
+      ++ lib.optionals (stdenvNoCC.buildPlatform.is32bit || true) [
```

Comment:
The `|| true` here looks unintentional, but it seems to me there's no reason for this to be conditional anyway.

Reply:
Gah, good catch! I made it unconditional.